### PR TITLE
Batch link related works

### PR DESCRIPTION
**Pull request by goose-life**

### CODE REVIEW

**Comment by goose-life** [comment]
closes #435

File: indigo_app/templates/indigo_api/work_new_batch.html [hunk 1]
```diff
@@ -110,7 +110,8 @@ <h5>Tasks for all works</h5>
 
         </div>
 
-        <label for="{{ form.workflows.id_for_label }}" class="h5">Add the tasks to the following workflows</label>
+        <label for="{{ form.workflows.id_for_label }}" class="h5">Workflows</label>
```
Comment (longhotsummer) [inline indigo_app/templates/indigo_api/work_new_batch.html]: If this is "less optional" than the tasks, perhaps move it above the tasks?

File: indigo_app/views/works.py [hunk 2]
```diff
@@ -559,6 +560,16 @@ def get_works(self, table):
                         info['status'] = 'success'
                         info['work'] = work
 
+                        # TODO: neaten this up
+                        if not row.get('commenced_by') == u"":
```
Comment (longhotsummer) [inline indigo_app/views/works.py]: how about `if row.get('commenced_by'):` because non-empty strings are thruthy, empty strings are falsey.

**Comment by longhotsummer** [comment]
@goose-life when looking up the linked works, I think we should filter by country. There's a good chance of works with matching titles in other countries.

File: indigo_app/views/works.py (hunk 2 above)
Comment (goose-life) [reply indigo_app/views/works.py]: of course, thanks – I initially just had it as `info['commenced_by'] = row.get('commenced_by')` and relied on this to be `None` or similar lower down, but that didn't work, so I went overboard in the other direction

**Pull request merged**